CoreWidgetProvider/.gitattributes

```diff
@@ -0,0 +1,13 @@
+# Set default behavior to automatically normalize line endings.
+
+* text=auto
+
+*.cs text eol=crlf
```

Comment:
Can these settings be changed in VS as well?  If so, would VS override these settings?

Reply:
Yes, you could use VS to change these, but my understanding of what gitattributes does is make sure the line endings are set on checkin/checkout. So you could change the line endings while you work, but it doesn't matter.